CMakeLists.txt

```
@@ -454,7 +454,7 @@ if(BUILD_TORCH)
     file(TO_NATIVE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/tensoradapter/pytorch/build.sh BUILD_SCRIPT)
     add_custom_target(
       tensoradapter_pytorch
-      ${CMAKE_COMMAND} -E env
+      ${CMAKE_COMMAND} -E env ROCM_PATH=${ROCM_PATH} CC=${CC} CXX=${CXX}
```

Let's keep this consistent

```suggestion
      ${CMAKE_COMMAND} -E env
      ROCM_PATH=${ROCM_PATH}
      CC=${CC}
      CXX=${CXX}
```

---

Doesn't look resolved to me?